libats-messaging-datatype/src/main/scala/com/advancedtelematic/libats/messaging_datatype/DataType.scala

```diff
@@ -77,4 +77,11 @@ object DataType {
                          deviceTime: Instant,
                          receivedAt: Instant,
                          payload: Json)
+
+  case class CorrelationId(id: String) extends AnyVal
```

Comment:
Let's not use this yet. Better to have proper urn class.

Reply:
Couldn't we just use this one in the meantime? I mean, we _are_ using this now in director and device-registry. At least we share the implementation and in the future when we refactor it we only need to change it here.